src/pages/[locale]/[...path]/index.md.astro

```diff
@@ -0,0 +1,16 @@
+---
+import ToMarkdown from '@components/ToMarkdown/ToMarkdown.astro';
+import Page from './index.astro';
```

Comment:
I like this pattern where the `Page` from `index.astro` can remain unmodified and this route does all the `ToMarkdown` magic. If a developer using Head Start doesn't need this behaviour, this `index.md.astro` route can simply be removed.